StarterProject/gulpfile.js

```diff
@@ -0,0 +1,6 @@
+
+var gulp = require("gulp");
+
+gulp.task("default", function() {
+    // place code for your default task here
+});
```

Comment:
What's the plan for gulp?  I don't really know what it would be used for (very little familiarity in general, not sure if it adds value for this project specifically)

Reply:
I will remove gulp from there